config/ember-try.js

```diff
@@ -1,2 +1,60 @@
+/*jshint node:true*/
 module.exports = {
```

Comment:
e-b uses the "versionCompatibility" feature of ember-try (https://github.com/ember-cli/ember-try#versioncompatibility) to check for all supported ember versions. So there is no need to specify any scenarios here. Therefore also the changes to `.travis.yml` should be reverted.